doc-surrealdb_versioned_docs/version-latest/surrealql/functions/database/index.mdx

```diff
@@ -9,102 +9,125 @@ import Since from '@site/src/components/Since'
 
 # Database Functions
 
-SurrealDB comes with a large number of in-built functions for checking, manipulating, and working with many different types of data. These functions are grouped into a number of different packages, which can be seen below.
+SurrealDB has many built-in functions designed to handle many common database tasks and work with SurrealDB's various data types, grouped into modules based on their purpose and the data types they are designed to work with. The table below lists all of SurrealDB's function modules, with descriptions and links to their own detailed documentation.
 
 <table>
   <thead>
     <tr>
-      <th scope="col">Module</th>
-      <th scope="col">Example / Notes</th>
+      <th scope="col" style="width:15%">Function Module</th>
+      <th scope="col" style="width:50%">Description</th>
+      <th scope="col" style="width:35%">Example</th>
     </tr>
   </thead>
   <tbody>
     <tr>
-      <td scope="row" data-label="Module"><a href="/docs/surrealdb/surrealql/functions/database/array"><code>Array Functions</code></a></td>
+      <td scope="row" data-label="Module"><a href="/docs/surrealdb/surrealql/functions/database/array"><code>Array</code></a></td>
+      <td scope="row" data-label="Description">These functions can be used when working with, and manipulating arrays of data.</td>
       <td scope="row" data-label="Example"><code>array::len([1,2,3])</code></td>
     </tr>
     <tr>
-      <td scope="row" data-label="Module"><a href="/docs/surrealdb/surrealql/functions/database/bytes"><code>Bytes functions</code></a></td>
+      <td scope="row" data-label="Module"><a href="/docs/surrealdb/surrealql/functions/database/bytes"><code>Bytes</code></a></td>
+      <td scope="row" data-label="Description">These functions can be used when working with bytes in SurrealQL.</td>
       <td scope="row" data-label="Example"><code>bytes::len("SurrealDB".to_bytes());</code></td>
     </tr>
     <tr>
-      <td scope="row" data-label="Module"><a href="/docs/surrealdb/surrealql/functions/database/count"><code>Count function</code></a></td>
+      <td scope="row" data-label="Module"><a href="/docs/surrealdb/surrealql/functions/database/count"><code>Count</code></a></td>
+      <td scope="row" data-label="Description">These functions can be used when counting field values and expressions.</td>
```

Comment:
Small note here: count is just a single function all on its own.

Reply:
@Dhghomon Thanks, I just corrected this in the table of all function modules, and on the documentation page of the `Count` module.